infrastructure/Taskfile.yml

```diff
@@ -1085,6 +1085,21 @@ tasks:
       preconditions:
       - *require_site
 
+    site:environment:sync:
+      desc: |
+        Synchonizes all file and database tables from environment
+        to another on make on environment a complete copy of the source.
+      deps: [lagoon:cli:config]
+      dir: "{{.dir_env}}"
+      vars:
+        SOURCE_PROJECT: "{{.SOURCE_PROJECT}}"
+        SOURCE_ENV: "{{.SOURCE_ENV}}"
+        TARGET_PROJECT: "{{.TARGET_PROJECT}}"
```

Comment:
Does this approach work across projects?

I wonder if `drush rsync` has the necessary configuration availabele to synchronize between different projects. In any case we might consider only allowing a single `PROJECT` variable.

Reply:
I have not tested wether this works across projects. It is not intended for that right now